asab/storage/mongodb.py

```diff
@@ -12,6 +12,7 @@
 		'asab:storage': {
 			'mongodb_uri': 'mongodb://localhost:27017',
 			'mongodb_database': 'asabdb',
+			'timezone_aware': 'yes',
```

Comment:
That should not be configurable - it will lead to diverse and complicated ecosystem.